src/index.ts

```diff
@@ -161,13 +129,9 @@ export class CdktfProviderProject extends cdk.JsiiProject {
         name: "team-tf-cdk",
         email: "github-team-tf-cdk@hashicorp.com",
       },
-      // sets major version to 1 for the first version but resets it for future versions to allow them to automatically increase to e.g. v2 if breaking changes occurred
-      majorVersion: options.forceMajorVersion ?? getMajorVersion(repository),
+      minMajorVersion: 1, // ensure breaking change releases lead to an increased major version
```

Comment:
nit (if you want)
```suggestion
      minMajorVersion: 1, // ensure new projects start with 1.0.0 so that every following breaking change leads to an increased major version
```